.github/workflows/bsim.yaml

```diff
@@ -0,0 +1,85 @@
+name: Bluetooth Tests
+
+on:
+  pull_request_target:
+    paths:
+      - "west.yml"
+      - "subsys/bluetooth/**"
+      - "tests/bluetooth/bsim_bt/**"
+      - "boards/posix/**"
+      - "soc/posix/**"
+      - "arch/posix/**"
+
+jobs:
+  bsim-build-cancel:
+    runs-on: ubuntu-latest
```

Comment:
Isn't it generally better to use a specific version to avoid updates breaking something?

Reply:
yes, but we get lots of warning before this happens, and that does not happen very often. If we change this, need to do that everywhere in our actions. For now, lets leave it like this and do a global change in some other PR.